docs/reference/experiment-config.txt

```diff
@@ -605,6 +605,13 @@ created with the Determined CLI. It may contain the following fields:
     ``tensor_fusion_threshold`` and ``tensor_fusion_cycle_time`` automatically.
     Defaults to ``false``.
 
+  - ``horovod_timeline``: The path where the Horovod should be timeline saved. The
```

Comment:
"The path where the Horovod timeline should be timeline".

Reply:
Also, mention that this is a container path? Or more generally should we talk about how we expect users to access / make use of the trace file? Maybe we can link to the upstream Horovod docs?

Reply:
Naming-wise, do we want to have `horovod` in the name of this config file option? I don't feel super strongly, although we don't use "Horovod" for other config file options.

Reply:
Fixed the typo and clarified the statement about the path. Because this feature is unpolished, I don't really want to make it something that average users try to use. IMO if you don't already know how to use the trace file or are working with us directly, you basically shouldn't use this flag. My thought was that a lack of detailed documentation would encourage this - curious what you think about this @neilconway.

I don't love having `horovod` in the name, but the target user is someone who understands Horovod pretty well and is specifically looking for how to generate the Horovod timeline, so I feel like it's good to use that standard term.